docs/installation.md

```diff
@@ -18,16 +18,10 @@ can guide you through the process.
 
 The sources for `watchmaker` can be downloaded from the [`GitHub repo`][2].
 
-You can either clone the public repository:
+First clone the public repository:
 
 ```bash
-git clone https://github.com/plus3it/watchmaker.git
-```
-
-Or download the [`tarball`][3]:
-
-```bash
-curl -OL https://github.com/plus3it/watchmaker/tarball/develop
+git clone https://github.com/plus3it/watchmaker.git && cd watchmaker
```

Comment:
D'oh. I should have expanded the collapsed lines and I would've noticed it.  My bad!